test/commands/info.js

```diff
@@ -82,6 +82,7 @@ describe('pg', () => {
       .then(() => expect(cli.stdout, 'to equal', `=== DATABASE_URL, HEROKU_POSTGRESQL_COBALT_URL
 Plan:      Hobby-dev
 Following: HEROKU_POSTGRESQL_COBALT
+App:       myapp2
```

Comment:
App as in billing app? Should we make that more explicit here? I'm on the fence, but it's not super clear.